### ISSUE TITLE
Descriptions for `find-X` commands can be misleading

### ISSUE THREAD

**Issue by gavingoh99** [gavingoh99/pe#13]
![Screenshot 2024-04-19 at 5.31.45 PM.png](https://raw.githubusercontent.com/gavingoh99/pe/main/files/a202b93d-6fc3-4ecd-a7f2-3f5ba6089732.png)

![Screenshot 2024-04-19 at 5.31.51 PM.png](https://raw.githubusercontent.com/gavingoh99/pe/main/files/98b2de55-82d4-4234-8b6a-bb457fc82932.png)

In the UG, the `find-X` commands are described as providing to ability to find clients whose names / tags **contain** any of the given keywords.

However, it is later specified that only full words are matched e.g `dis` will not match `disabled`. In layman English, a client with the tag `disabled` does actually contain the keyword `dis` so we would expect this client to match!

Given the conflicting descriptions, it could be better if the descriptions were changed to `match` instead of `contain`!

**Comment by nus-pe-script** [comment]
# Team's Response
"contain any of the given KEYWORDS" implies that whole / full words will be used to match as keywords means "a WORD or concept of great significance", not a part of a word.


![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S2/pe/master/files/c51d0b6c-a6fb-49e0-a572-570cd5c69d4d.png)


We later even specified that **only full words** are matched e.g dis will not match disabled.

Since the functionality is accurately described in the detailed section of the documentation, we will not classify this as a bug.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** Thanks for pointing out that keywords should refer to whole words.

Let me posit another example with respect to this:

Suppose a user tags a client with the tag `HasADog` and decides to search by tag. In this context, surely the word `dog` is a **whole** word and also one of **great significance**. However, SWEE will not match the query with the contact because it does not support substring search.

Again, I'm not disputing that the functionality has been described further down and I've even acknowledged this in my own report. What I'm getting at is that if I were a user in a hurry and only read the first short description under the command, I would be inclined to think that as long as the tag contains the keyword we should expect that SWEE will return said client. The misalignment between this short description and the detailed description can potentially lead to confusion and thus that's why I believe it to be a documentation bug.

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.VeryLow`]
Originally [`severity.Low`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
